internal/importers/glide/glide_importer_test.go

```diff
@@ -225,16 +211,16 @@ func TestGlideConfig_Import(t *testing.T) {
 		t.Fatal("Expected the lock to be generated")
 	}
 
-	goldenFile := "init/glide/golden.txt"
+	goldenFile := "golden.txt"
 	got := verboseOutput.String()
-	want := h.GetTestFileString(goldenFile)
-	if want != got {
+	Want := h.GetTestFileString(goldenFile)
```

Comment:
unnecessary capitalization - makes me wonder if there might be a few more of these that i've missed?

Reply:
Doh! My find/replace was a bit too greedy! I'll find the rest.